controller-api/src/main/java/com/yahoo/vespa/hosted/controller/api/integration/noderepository/NodeRepositoryNode.java

```diff
@@ -88,6 +88,8 @@
     private String modelName;
     @JsonProperty("reservedTo")
     private String reservedTo;
+    @JsonProperty("exclusiveTo")
+    private NodeOwner exclusiveTo;
```

Comment:
It's a string on `ApplicationId::serializedForm` format, e.g.: https://github.com/vespa-engine/vespa/blob/70c61f3efe2e4e761c5966db6926a6a4c008aec0/node-repository/src/test/java/com/yahoo/vespa/hosted/provision/restapi/responses/parent2.json#L10

Reply:
Fixed